hr_attendance_management/models/hr_holidays.py

```diff
@@ -37,7 +37,7 @@ def _compute_att_day(self):
                 ]
             )
 
-            rd.sudo().attendance_day_ids = att_days
+            self.attendance_day_ids = att_days
```

Comment:
I don't think this change is valid. Why would you change the whole recordset at each iteration? Can you test if your fix works by reverting this change and only keeping the next at line 55? Otherwise please explain me what is changed here.